### PR TITLE
Fix deprecation warnings

### PR DESCRIPTION
**Pull request by pangratz**
I kept each commit separately to see which deprecations are fixed how. If this is :+1:, I will squash them and prefix with `[CLEANUP beta]`.

### CODE REVIEW

**Comment by wecc** [comment]
LGTM

File: tests/index.html [hunk 1]
```diff
@@ -62,7 +62,7 @@
 
         var skipPackageRegexp = new RegExp('^('+skipPackages.join('|')+')/');
 
-        var keys = Ember.keys(requirejs.entries);
+        var keys = Object.keys(requirejs.entries);
```
Comment (pangratz) [inline tests/index.html]: This one has been missed in #3432

Comment (wecc) [inline tests/index.html]: Nice catch!

**Comment by bmac** [comment]
Looks good.

**Pull request merged**